### ISSUE TITLE
Some question about IRM.

### ISSUE THREAD

**Issue by Ma-Zijing** [facebookresearch/InvariantRiskMinimization#8]
Hi lopezpaz, IRM is a fantastic work and I am reading and trying to reproduce it. But I meet some questions.

1. When it comes to the colored MINIST experiment. The paper says it divides the MINIST into 3 environments and colors the images to red and green. So Does it mean the test environment also contains digits of the same color in the training set? I mean, suppose that there are digits 9 with green color, and the test set also contains this kind of images?

2. Can IRM solve multiple classification problems? If yes, should I replace binary loss with cross-entropy loss?

I am looking forward to you reply! Thanks.
